metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/indexing_master.py

```diff
@@ -115,6 +116,16 @@ def elasticsearch_template_delete(self, env):
         yaf_cmd = ambari_format('curl -s -XDELETE "http://{es_http_url}/yaf_index*"')
         Execute(yaf_cmd, logoutput=True)
 
+    def zeppelin_dashboard_install(self, env):
+        from params import params
+        env.set_params(params)
+
+        for dirName, subdirList, files in os.walk(params.metron_config_zeppelin_path):
```

Comment:
I'll add logging for that.  You're right, in the case that there are no notebooks, it'd be helpful to see what it's using.